src/uniprotkb/components/entry/Entry.tsx

```diff
@@ -201,7 +231,23 @@ const Entry = () => {
             );
             break;
           default:
-            disabled = !hasContent(transformedData[nameAndId.id]);
+            disabled =
+              !hasContent(transformedData[nameAndId.id]) &&
+              !communityReferences.some((reference) => {
+                if (
+                  reference?.communityAnnotation &&
+                  (nameAndId.id === EntrySection.Function ||
+                    nameAndId.id === EntrySection.NamesAndTaxonomy ||
+                    nameAndId.id === EntrySection.DiseaseVariants ||
+                    nameAndId.id === EntrySection.PhenotypesVariants)
+                ) {
```

Comment:
If you're happy to make `entrySectionToCommunityAnnotationField` this could be used here too:
```
entrySectionToCommunityAnnotationField.has(nameAndId.id)
```